core/arch/arm/mm/core_mmu.c

```diff
@@ -158,8 +158,9 @@ static void core_mmu_mmap_init(struct tee_mmap_region *mm, size_t max_elem,
 
 	for (n = 0; map[n].type != MEM_AREA_NOTYPE; n++) {
 		mme.pa = map[n].pa;
-		mme.va = map[n].pa;
+		mme.va = map[n].va;
```

Comment:
@pascal-brand-st-dev 
I introduce an entry `struct map_area *map; /* The map that mmap region refer to */` for  `struct tee_mmap_region`, then there is no need for mme.va/pa/size, will discard them in v2 patch.